Sources/Apollo/ApolloClient.swift

```diff
@@ -133,23 +127,15 @@ extension ApolloClient: ApolloClientProtocol {
   public func clearCache(callbackQueue: DispatchQueue = .main, completion: ((Result<Void, Error>) -> Void)? = nil) {
     self.store.clearCache(completion: completion)
   }
-
+  
   @discardableResult public func fetch<Query: GraphQLQuery>(query: Query,
-                                                            cachePolicy: CachePolicy = .returnCacheDataElseFetch,
-                                                            context: UnsafeMutableRawPointer? = nil,
-                                                            queue: DispatchQueue = DispatchQueue.main,
-                                                            resultHandler: GraphQLResultHandler<Query.Data>? = nil) -> Cancellable {
-    let resultHandler = wrapResultHandler(resultHandler, queue: queue)
-
-    // If we don't have to go through the cache, there is no need to create an operation
-    // and we can return a network task directly
-    if cachePolicy == .fetchIgnoringCacheData || cachePolicy == .fetchIgnoringCacheCompletely {
-      return self.send(operation: query, shouldPublishResultToStore: cachePolicy != .fetchIgnoringCacheCompletely, context: context, resultHandler: resultHandler)
-    } else {
-      let operation = FetchQueryOperation(client: self, query: query, cachePolicy: cachePolicy, context: context, resultHandler: resultHandler)
-      self.operationQueue.addOperation(operation)
-      return operation
-    }
+                                                                     cachePolicy: CachePolicy = .returnCacheDataElseFetch,
+                                                                     context: UnsafeMutableRawPointer? = nil,
```

Comment:
I'm not sure if that is related to this PR, but in the `perform(mutation:)` function, the `context: UnsafeMutableRawPointer? = nil,` was removed, is there a reason why it is kept in the fetch function?

Reply:
nope, good catch - it should stay in the store for direct interaction with the store, but otherwise you're correct.